doc/GIT_USAGE.md

```diff
@@ -19,23 +19,23 @@
 ## Branching Model
 
 - The DMOD repo uses a branching model based on [Gitflow](https://nvie.com/posts/a-successful-git-branching-model/) and has two primary long-term branch: 
-  - **master**: the main branch pointing to the current, official version
-  - **dev**: an integration branch containing the latest completed development work intended for the next released version
-- Most interaction with DMOD repo is done via pull requests (PRs) to the `dev` branch
-  - Independent branches for features or bug fixes are created off `dev` to contain development work that is in progress
-  - These branches are reviewed and their changes integrated back into `dev` once complete via PRs
+  - **master**: the main development and integration branch containing the latest completed development work intended for the next released version
+  - **production**: the branch representing the latest code verified as production-ready and pointing to the most recently release, official version
+- Most interaction with DMOD repo is done via pull requests (PRs) to the `master` branch
+  - Independent branches for features or bug fixes are created off `master` to contain development work that is in progress
+  - These branches are reviewed and their changes integrated back into `master` once complete via PRs
   - Typically feature/fix branches exist in personal clones and personal Github forks, but not in the official OWP repo
 - Release branches (e.g., `release-X` for pending version `X`) will periodically be created to finalize the next new version when it is time for it to be released
```

Comment:
Can you clarify this by stating whether the release branch will be created _every_ release or as needed?

Reply:
Sure, I will reword things.

The intent was for a release branch to be created every time there was a new release, to provide a clear, isolated target for testing, QA, etc.  But what exactly constitutes "testing, QA, etc." is intentionally undefined for the time being, as is how to decide when it is time to cut the next release.